### PR TITLE
Fix zwave_js set config dropdown default value

### PR DESCRIPTION
**Pull request by cgarwood**
## Proposed change

With the switch from paper elements to `<ha-select>` the default value wasn't working in the Z-Wave JS Node Config panel. From what I could tell, the `value` was sometimes being parsed as an integer, while the 
`value` of a list-item was being parsed as a string. So we're coercing all the values to a string.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by kpine** [comment]
Fixes https://github.com/home-assistant/core/issues/67765.

File: src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts [hunk 1]
```diff
@@ -345,7 +345,10 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
     if (ev.target === undefined || this._config![ev.target.key] === undefined) {
       return;
     }
-    if (this._config![ev.target.key].value === ev.target.value) {
+    if (
+      this._config![ev.target.key].value?.toString() ===
+      ev.target.value.toString()
```
Comment (balloob) [inline src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts:350]: isn't it already a string?
```suggestion
      ev.target.value
```

Comment (cgarwood) [reply src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts]: Good point.

**Review by zsarnett — DISMISSED** [review]
Needs to run prettier

**Review by zsarnett — APPROVED** [review]

**Pull request merged**